docscan/session/create/import_token.go

```diff
@@ -0,0 +1,35 @@
+package create
+
+import "time"
+
+type ImportToken struct {
+	Ttl int `json:"ttl"`
+}
+
+// defaultImportTokenTTL is 12 Months
+const defaultImportTokenTTL = time.Hour * 24 * 30 * 12
```

Comment:
12 months seems like a very long time, i'd make the default 1 month so we're recommending security (by giving an attacker less time) and only the RP can change it if they need it to be longer.

Reply:
oh also i wouldn't say that's accurately 12 months, if you wanna do a year i would've just done `time.Hour * 24 * 365` (rip leap years)

Reply:
in the ticket we're defining ttl as: "seconds, not null, default to 12 months, MAX 12 months, not less than 30 days"